scripts/individual-packages.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
How about calling this `transform.sh`.

The architecture of our repo will be build → transform → pack.

One (and currently only) step in 'transform' is the transformation for v2 packages.

Reply:
Done.